File: java/src/test/java/com/testexercise/ebayframework/CartItemWidget.java
package com.testexercise.ebayframework;

import org.openqa.selenium.*;

public class CartItemWidget extends BaseWidget
{
    public CartItemWidget(WebElement elem)
    {
        super(elem);
    }
    
    private final String cartItemLinkCss = "div.fl.infocolcart140 a";
    private final String cartItemPriceCss = "div.fw-b";
    
    public ItemDetails getItemDetails()
    {
        WebElement link = boundingElem.
                findElement(By.cssSelector(cartItemLinkCss));
        
        WebElement price = boundingElem.
                findElement(By.cssSelector(cartItemPriceCss));
        
        return new ItemDetails(
                link.getText().trim(),
                link.getAttribute("href"),
                price.getText());
    }
}

File: net/TestExerciseEbay/EbayPOMFramework/CartItemWidget.cs
using System;
using OpenQA.Selenium;

namespace TestExercise.EbayPOMFramework
{
    public class CartItemWidget : BaseWidget
    {
        public CartItemWidget(IWebElement elem) : base(elem) { }


        private const string cartItemLinkCss = "div.fl.infocolcart140 a";
        private const string cartItemPriceCss = "div.fw-b";

        public ItemDetails GetItemDetails()
        {
            IWebElement link = boundingElem.
                    FindElement(By.CssSelector(cartItemLinkCss));

            IWebElement price = boundingElem.
                    FindElement(By.CssSelector(cartItemPriceCss));

            return new ItemDetails(
                    link.Text.Trim(),
                    link.GetAttribute("href"),
                    price.Text);
        }
    }
}